src/qr/qr.controller.ts

```diff
@@ -0,0 +1,58 @@
+import { paths } from '@/generated/schema';
+import {
+	Controller,
+	Get,
+	NotFoundException,
+	Param,
+	ParseUUIDPipe,
+} from '@nestjs/common';
+import { ArtsService } from '@/arts/arts.service';
+
+@Controller('qr')
+export class QrController {
+	constructor(private readonly artsService: ArtsService) {}
+	/**
+	 * QRを読み込み時にAR作品のオブジェクト格納URL等を返す。
+	 * @returns paths['/api/v1/qr/{art_id}']['get']['responses']['200']
+	 */
+
+	@Get(':art_id')
+	async getArt(
+		/**
+		 * parametersで`art_id`取得
+		 * paths['/api/v1/qr/{art_id}']['get']['parameters']['path']['art_id']
+		 */
+		@Param(
+			'art_id' satisfies paths['/api/v1/qr/{art_id}']['get']['parameters']['path']['art_id'],
+			new ParseUUIDPipe(),
+		)
+		art_id: string,
+	): Promise<any> {
+		const qrArtDetail = await this.artsService.getQr(art_id);
+
+		/**
+		 * 作品が存在しない場合
+		 * @throws paths['/api/v1/qr/{art_id}']['get']['responses']['404']
+		 */
+		if (!qrArtDetail) {
+			throw new NotFoundException() satisfies paths['/api/v1/qr/{art_id}']['get']['responses']['404']['content']['application/json'];
+		}
+
+		const result = {
+			'arts.id': qrArtDetail.id,
+			'arts.name': qrArtDetail.name,
+			'arts.address': qrArtDetail.address,
+			'arts.description': qrArtDetail.description,
+			'arts.created_year': qrArtDetail.created_year,
+			'arts.datetime_description': qrArtDetail.datetime_description,
+			'arts.closed_day_description': qrArtDetail.closed_day_description,
+			'institutions.admission_fee_description':
+				qrArtDetail.arts_institutions[0].institution.admission_fee_description,
+			'authors.name': qrArtDetail.authors.name,
+			poster_path: qrArtDetail.poster_path,
+			glb_path: qrArtDetail.glb_path,
+			reality_path: qrArtDetail.reality_path,
+		} as const;
+		return result satisfies paths['/api/v1/qr/{art_id}']['get']['responses']['200']['content']['application/json'];
```

Comment:
```suggestion
		} as const satisfies paths['/api/v1/qr/{art_id}']['get']['responses']['200']['content']['application/json'];
		return result;
```

修正不要、こっちのが綺麗な気もする、宗教